tala_connector.py

```diff
@@ -262,9 +265,9 @@ def _handle_create_project(self, param):
         summary = action_result.update_summary({})
         summary['project_id'] = response['id']
 
-        # Return success, no need to set the message, only the status
+        # Return success status with appropriate message
         # BaseConnector will create a textual message based off of the summary dictionary
-        return action_result.set_status(phantom.APP_SUCCESS)
+        return action_result.set_status(phantom.APP_SUCCESS, TALA_CREATED_PROJECT_SUCC)
```

Comment:
Please revert this change. We are setting `project_id` in the summary, hence, action_resilt's message will not be empty.

Reply:
Reverted.